pkg/kamailio/obs/kamailio.spec

```diff
@@ -132,21 +139,27 @@
 %if 0%{?rhel} == 7 && 0%{?centos_ver} != 7
 %define dist_name rhel
 %define dist_version %{?rhel}
-%bcond_without cnxcc
+%bcond_with cnxcc
 %bcond_with dnssec
-%bcond_without geoip
-%bcond_without http_async_client
-%bcond_without jansson
-%bcond_without json
-%bcond_without kazoo
-%bcond_without memcached
+%bcond_with geoip
+%bcond_with http_async_client
+%bcond_with jansson
+%bcond_with json
+%bcond_with kazoo
+%bcond_with memcached
 %bcond_without perl
+%bcond_without rebbitmq
 %bcond_without redis
-%bcond_without sctp
-%bcond_without websocket
+%bcond_with sctp
+%bcond_with websocket
 %bcond_without xmlrpc
```

Comment:
This reverts 612e96d8a2cc2a9ac31a7fb05508084fb82dbcf5. Unintentional?

Reply:
What repo you used to build `sctp` and `websocket` modules on `RHEL`?

Reply:
Created PR https://github.com/kamailio/kamailio/pull/1317

Reply:
Both libunistring-devel and lksctp-tools-devel are in the rhel7-server-optional repo, which my machine has enabled by default.

When bulding packages in EPEL they enable the same repo, ref
http://fedoraproject.org/wiki/EPEL/GuidelinesAndPolicies#Policy

Reply:
Could you help find way how to attache this optional repo on OpenSUSE build server.
I see OBS not provide this optional repo.
Without this optional repo we cannot build this packages on available infrastructure.

Reply:
With a quick search I found some details at:

  * https://access.redhat.com/documentation/en-us/red_hat_subscription_management/1/html/rhsm/supplementary-repos

But not sure if that command can be used in OBS.

Reply:
yes Daniel
This is requires active RedHat support contact.

Reply:
I think add instruction how to build this modules to README file for RHEL dist.

Reply:
@oyvindkolbu Please test instructions on commit https://github.com/kamailio/kamailio/pull/1316/commits/35f28d741f485a8b6b6ffbfc202844254f696790
And let me know, please.